assets/js/blocks-registry/payment-methods/payment-method-config.js

```diff
@@ -86,5 +87,14 @@ export default class PaymentMethodConfig {
 				'If the payment method includes the `supports.savePaymentInfo` property, it must be a boolean'
 			);
 		}
+		if (
+			config.supports &&
+			typeof config.supports.requiresSaving !== 'undefined' &&
```

Comment:
If you use optional chaining here you can get rid of the previous line.

Reply:
Great idea, let's add this as a cooldown task to tidy this up so we don't pollute the PR. Thanks for the tip!

Reply:
Never mind, I see how small this change is, I thought it was going to be much larger. I've added it to the PR!